src/clj_commons/leiningen/vizdeps.clj

```diff
@@ -313,3 +321,56 @@
     (let [dot (build-dot project options)]
       (common/write-files-and-view dot options)
       (main/info "Wrote dependency chart to:" (:output-path options)))))
+
+
+
+(defn- insecure-http-abort [& _]
+  (let [repo (promise)]
+    (reify org.apache.maven.wagon.Wagon
+      (getRepository [this])
+      (setTimeout [this _])
+      (setInteractive [this _])
+      (addTransferListener [this _])
+      (^void connect [this
+                      ^org.apache.maven.wagon.repository.Repository the-repo
+                      ^org.apache.maven.wagon.authentication.AuthenticationInfo _
+                      ^org.apache.maven.wagon.proxy.ProxyInfoProvider _]
+       (deliver repo the-repo) nil)
+      (get [this resource _]
+        (main/abort "Tried to use insecure HTTP repository without TLS:\n"
+                    (str (.getId @repo) ": " (.getUrl @repo) "\n " resource) "\n"
+                    "\nThis is almost certainly a mistake; for details see"
+                    "\nhttps://github.com/technomancy/leiningen/blob/master/doc/FAQ.md")))))
+
+
+(defn- configure-http
```

Comment:
What do we get from copying over these functions out of Leiningen? Is it just so that we don't get broken if Leiningen changes this API between versions?